lib/get-crypto.ts

```diff
@@ -0,0 +1,9 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
+import { webcrypto } from "crypto" //Node equivalent of window.crypto in browser
+let cryptoExport: webcrypto.Crypto;
+if (webcrypto == undefined && window !== undefined) {
+    cryptoExport = window.crypto as webcrypto.Crypto;
+} else {
+    cryptoExport = webcrypto;
+}
+export default cryptoExport 

```

Comment:
Hmmm, so importing the library will automatically register this to window, yes?

Reply:
window.crypto is a browser feature, present in all browsers by default (no import necessary). https://developer.mozilla.org/en-US/docs/Web/API/Crypto

The import is here for nodejs. When using webpack for browser, the import will result in undefined, thus trying to get it from window.crypto.